src/fides/core/export.py

```diff
@@ -193,7 +193,6 @@ def generate_system_records(  # pylint: disable=too-many-nested-blocks, too-many
             custom_columns[key_string] = value
             keys.append(key_string)
             output_list[0] = tuple(keys)
-
```

Comment:
wait @SteveDMurphy i think ya may have meant to remove the line below, no?

Reply:
Yep! 🤦🏽 blame it on the hour, thanks for catching